### ISSUE TITLE
Output widget does not work in notebook 4.x

### ISSUE THREAD

**Issue by jdemeyer** [ipython/ipywidgets#1050]
Doing exactly what the documentation of `Output?` says does not actually work. When creating a notebook cell with the code
```python
import ipywidgets as widgets
from IPython.display import display
out = widgets.Output()
display(out)

print('prints to output area')

with out:
    print('prints to output widget')
```
the string `prints to output widget` is displayed normally and ''not'' in the `Output` widget.

**Comment by jasongrout** [comment]
Output widget should work if you have one of the recent ipywidgets/widgetsnbextension beta versions, and you are either on notebook master or you're using https://github.com/jupyter/notebook/pull/2022.

Or if you're on jlab.

**Comment by jasongrout** [comment]
Output widget is I think completely broken in ipywidgets on released versions of notebook 4.x.

**Comment by jasongrout** [comment]
To resolve this, we should either:

(a) decide that we only need to support notebook 5.0, and have 5.0 released before ipywidgets 6.0.

or 

(b) release notebook 4.4 with my changes at https://github.com/jupyter/notebook/pull/2022 to enable the output widget

@SylvainCorlay - this impacts our release timeline for 6.0.

**Comment by SylvainCorlay** [comment]
I would like to get https://github.com/jupyter/notebook/issues/2087 fixed in notebook 5.0.

**Comment by jasongrout** [comment]
We've now merged the necessary changes and plan to release Notebook 4.4 supporting the output widget.
